pyproject.toml

```diff
@@ -45,6 +45,7 @@ sphinx_rtd_theme = {version=">=1.0", optional=true}
 grpcio = {version=">=1.49.0,<2.0", optional = true}
 protobuf = { version = "^4.21", optional = true }
 hightime = "^0.2.1"
+tzlocal = "^5.2"
```

Comment:
`^5.2` means `>=5.2,<6.0`. If you want to use `nidaqmx` with `tzlocal` 5.1, too bad, you're not allowed. Likewise, if you want to use `nidaqmx` with `tzlocal` 6.0 when it is released, you can't.

So I would normally recommend specifying a looser constraint like `>=5.0`.

Looking at `tzlocal`'s version history to see how far back we can go, I see it broke compatibility in 3.0, reverted it in 4.0, and broke it again in 5.0.
https://github.com/regebro/tzlocal?tab=readme-ov-file#api-change

So I guess it's safe to assume that a `tzlocal` 6.0 would break compatibility, so we should use `^5.0`

Reply:
Reverted my tzlocal 5.2 change, updated it to be ^5.0 and reran poetry.lock.